YumemiInternTask/Contract/WeatherContract.swift

```diff
@@ -16,6 +20,10 @@ protocol WeatherPresentationOutput: AnyObject {
     /// 取得した天気を出力する
     /// - parameter weatherString;天気を表す文字列
     func outputWeather(weatherString: String)
+
+    /// エラーを出力する
+    /// - parameter error:エラー
+    func outputWeatherError(error: WeatherError)
```

Comment:
ここも使うときに `outputWeatherError(error: hoge)` ってなって error がだぶるので、

```suggestion
    func outputWeatherError(with error: WeatherError)
```

とかかな 🤔 

↑ 微妙な気もするので、考えてみてください

Reply:
考えてみましたが、
``func outputWeatherError(using error: WeatherError)``(これは微妙っぽい)
とか関数名からエラーを出力するのは明白なので
``func outputWeatherError(_ error: WeatherError)``
どうかなと思いましたが、どうでしょう？？

Reply:
良いと思います！